src/wp-admin/edit-form-advanced.php

```diff
@@ -626,27 +683,37 @@
  */
 do_action( 'edit_form_after_title', $post );
 
-if ( post_type_supports($post_type, 'editor') ) {
-?>
-<div id="postdivrich" class="postarea<?php if ( $_wp_editor_expand ) { echo ' wp-editor-expand'; } ?>">
-
-<?php wp_editor( $post->post_content, 'content', array(
-	'_content_editor_dfw' => $_content_editor_dfw,
-	'drag_drop_upload' => true,
-	'tabfocus_elements' => 'content-html,save-post',
-	'editor_height' => 300,
-	'tinymce' => array(
-		'resize' => false,
-		'wp_autoresize_on' => $_wp_editor_expand,
-		'add_unload_trigger' => false,
-		'wp_keep_scroll_position' => ! $is_IE,
-	),
-) ); ?>
+if ( post_type_supports( $post_type, 'editor' ) ) {
+	$_wp_editor_expand_class = '';
+	if ( $_wp_editor_expand ) {
+		$_wp_editor_expand_class = ' wp-editor-expand';
+	}
+	?>
+<div id="postdivrich" class="postarea<?php echo $_wp_editor_expand_class; ?>">
```

Comment:
It seems like sometimes the if statements are moved to before the output, and sometimes not. 
To me, it is more legible inline. It also takes less memory. Also, there is no function definition in this file, so the new variables have whatever scope from where this is included.  (_could be global variables_)

Reply:
I wonder if that is why the variable is underscored at the start.

Reply:
Discussed ar PR review